notebooks/ICESat-2_Cloud_Access/ICESat2-CryoSat2-Coincident.ipynb

```diff
@@ -436,7 +434,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.14"
+   "version": "3.9.18"
```

Comment:
This one actually downgraded from 3.10.14 to 3.9.18? Others went from 3.9.16 to 3.9.18 (?)